### ISSUE TITLE
(not an issue) The following environment.yml might help

### ISSUE THREAD

**Issue by ArghyaPal** [yiranran/Audio-driven-TalkingFace-HeadPose#96]
One may try to use the following to reproduce the code,

`name: <Your_environment_name>
channels:
  - pytorch
  - defaults
  - conda-forge
dependencies:
  - _libgcc_mutex=0.1=main
  - _openmp_mutex=5.1=1_gnu
  - _pytorch_select=0.2=gpu_0
  - _tflow_select=2.1.0=gpu
  - absl-py=0.15.0=pyhd3eb1b0_0
  - astor=0.8.1=py36h06a4308_0
  - audioread=2.1.9=py36h5fab9bb_0
  - blas=1.0=mkl
  - bzip2=1.0.8=h7b6447c_0
  - c-ares=1.18.1=h7f8727e_0
  - ca-certificates=2022.10.11=h06a4308_0
  - certifi=2021.5.30=py36h06a4308_0
  - cffi=1.14.6=py36h400218f_0
  - cudatoolkit=10.1.243=h6bb024c_0
  - cycler=0.11.0=pyhd3eb1b0_0
  - dataclasses=0.8=pyh4f3eec9_6
  - dbus=1.13.18=hb2f20db_0
  - expat=2.4.9=h6a678d5_0
  - ffmpeg=4.2.2=h20bf706_0
  - fontconfig=2.13.1=hef1e5e3_1
  - freetype=2.12.1=h4a9f257_0
  - gast=0.5.3=pyhd3eb1b0_0
  - gettext=0.21.0=hf68c758_0
  - glib=2.69.1=h4ff587b_1
  - gmp=6.2.1=h295c915_3
  - gnutls=3.6.15=he1e5248_0
  - google-pasta=0.2.0=pyhd3eb1b0_0
  - grpcio=1.31.0=py36hf8bcb03_0
  - gst-plugins-base=1.14.0=h8213a91_2
  - gstreamer=1.14.0=h28cd5cc_2
  - h5py=2.10.0=py36hd6299e0_1
  - hdf5=1.10.6=hb1b8bf9_0
  - icu=58.2=he6710b0_3
  - importlib-metadata=4.8.1=py36h06a4308_0
  - intel-openmp=2022.1.0=h9e868ea_3769
  - joblib=1.0.1=pyhd3eb1b0_0
  - jpeg=9e=h7f8727e_0
  - keras-applications=1.0.8=py_1
  - keras-preprocessing=1.1.2=pyhd3eb1b0_0
  - kiwisolver=1.3.1=py36h2531618_0
  - krb5=1.19.2=hac12032_0
  - lame=3.100=h7b6447c_0
  - lcms2=2.12=h3be6417_0
  - ld_impl_linux-64=2.38=h1181459_1
  - lerc=3.0=h295c915_0
  - libcurl=7.85.0=h91b91d3_0
  - libdeflate=1.8=h7f8727e_5
  - libedit=3.1.20210910=h7f8727e_0
  - libev=4.33=h7f8727e_1
  - libffi=3.3=he6710b0_2
  - libflac=1.3.4=h27087fc_0
  - libgcc-ng=11.2.0=h1234567_1
  - libgfortran-ng=7.5.0=ha8ba4b0_17
  - libgfortran4=7.5.0=ha8ba4b0_17
  - libgomp=11.2.0=h1234567_1
  - libidn2=2.3.2=h7f8727e_0
  - libnghttp2=1.46.0=hce63b2e_0
  - libogg=1.3.5=h27cfd23_1
  - libopus=1.3.1=h7b6447c_0
  - libpng=1.6.37=hbc83047_0
  - libprotobuf=3.17.2=h4ff587b_1
  - librosa=0.7.0=py_0
  - libsndfile=1.0.31=h9c3ff4c_1
  - libssh2=1.10.0=h8f2d780_0
  - libstdcxx-ng=11.2.0=h1234567_1
  - libtasn1=4.16.0=h27cfd23_0
  - libtiff=4.4.0=hecacb30_2
  - libunistring=0.9.10=h27cfd23_0
  - libuuid=1.41.5=h5eee18b_0
  - libuv=1.40.0=h7b6447c_0
  - libvorbis=1.3.7=h7b6447c_0
  - libvpx=1.7.0=h439df22_0
  - libwebp-base=1.2.4=h5eee18b_0
  - libxcb=1.15=h7f8727e_0
  - libxml2=2.9.14=h74e7548_0
  - llvmlite=0.29.0=py36hd408876_0
  - lz4-c=1.9.3=h295c915_1
  - markdown=3.3.4=py36h06a4308_0
  - matplotlib=3.3.4=py36h06a4308_0
  - matplotlib-base=3.3.4=py36h62a2d02_0
  - mkl=2020.2=256
  - mkl-service=2.3.0=py36he8ac12f_0
  - mkl_fft=1.3.0=py36h54f3939_0
  - mkl_random=1.1.1=py36h0573a6f_0
  - ncurses=6.3=h5eee18b_3
  - nettle=3.7.3=hbbd107a_1
  - ninja=1.10.2=h06a4308_5
  - ninja-base=1.10.2=hd09550d_5
  - numba=0.45.1=py36h962f231_0
  - numpy=1.19.2=py36h54aff64_0
  - numpy-base=1.19.2=py36hfa32c7d_0
  - olefile=0.46=py36_0
  - openh264=2.1.1=h4ff587b_0
  - openjpeg=2.4.0=h3ad879b_0
  - openssl=1.1.1s=h7f8727e_0
  - pcre=8.45=h295c915_0
  - pip=21.2.4=pyhd8ed1ab_0
  - protobuf=3.17.2=py36h295c915_0
  - pycparser=2.21=pyhd3eb1b0_0
  - pyparsing=3.0.4=pyhd3eb1b0_0
  - pyqt=5.9.2=py36h05f1152_2
  - pysoundfile=0.11.0=pyhd8ed1ab_0
  - python=3.6.13=h12debd9_1
  - python-dateutil=2.8.2=pyhd3eb1b0_0
  - python_abi=3.6=2_cp36m
  - pytorch=1.8.1=py3.6_cuda10.1_cudnn7.6.3_0
  - qt=5.9.7=h5867ecd_1
  - readline=8.2=h5eee18b_0
  - resampy=0.2.2=py_0
  - scikit-learn=0.24.2=py36ha9443f7_0
  - scipy=1.5.2=py36h0b6359f_0
  - setuptools=49.6.0=py36h5fab9bb_3
  - sip=4.19.8=py36hf484d3e_0
  - six=1.16.0=pyhd3eb1b0_1
  - sqlite=3.39.3=h5082296_0
  - tbb=2021.6.0=hdb19cb5_0
  - tensorboard=1.14.0=py36hf484d3e_0
  - tensorflow=1.14.0=hc3e5e64_0
  - tensorflow-base=1.14.0=py36hc3e5e64_0
  - tensorflow-estimator=1.14.0=py_0
  - tensorflow-gpu=1.14.0=h0d30ee6_0
  - termcolor=1.1.0=py36h06a4308_1
  - threadpoolctl=2.2.0=pyh0d69192_0
  - tk=8.6.12=h1ccaba5_0
  - torchvision=0.2.2=py_3
  - tornado=6.1=py36h27cfd23_0
  - typing_extensions=4.1.1=pyh06a4308_0
  - werkzeug=2.0.3=pyhd3eb1b0_0
  - wheel=0.37.1=pyhd3eb1b0_0
  - wrapt=1.12.1=py36h7b6447c_1
  - x264=1!157.20191217=h7b6447c_0
  - xz=5.2.6=h5eee18b_0
  - zipp=3.6.0=pyhd3eb1b0_0
  - zlib=1.2.13=h5eee18b_0
  - zstd=1.5.2=ha4553b6_0
  - pip:
    - beautifulsoup4==4.11.1
    - charset-normalizer==2.0.12
    - decorator==4.4.2
    - dlib==19.18.0
    - dominate==2.4.0
    - easydict==1.9
    - filelock==3.4.1
    - gdown==4.5.3
    - idna==3.4
    - imageio==2.15.0
    - importlib-resources==5.4.0
    - mxnet==1.5.1.post0
    - mxnet-cu101==1.5.0
    - networkx==2.5.1
    - opencv-python==4.1.0.25
    - pillow==8.4.0
    - pysocks==1.7.1
    - python-graphviz==0.8.4
    - python-speech-features==0.6
    - pywavelets==1.1.1
    - requests==2.27.1
    - scikit-image==0.15.0
    - soupsieve==2.3.2.post1
    - tqdm==4.64.1
    - urllib3==1.26.12
prefix: /<your_path>/anaconda3/envs/<your_environment>`
